src/scales/scale.linear.js

```diff
@@ -12,6 +12,31 @@ module.exports = function(Chart) {
 	};
 
 	var LinearScale = Chart.LinearScaleBase.extend({
+
+		scaleLimitSnapshot: {
```

Comment:
Since this is only used once, I think it could be returned from the `takeScaleLimit` function rather than storing it on  the axis